storybook/src/pages/amopis/DetailsPage/DetailsPage.tsx

```diff
@@ -0,0 +1,163 @@
+import {
+  AspectRatio,
+  Breadcrumb,
+  Column,
+  DescriptionList,
+  Grid,
+  Heading,
+  Image,
+  Link,
+  LinkList,
+  Paragraph,
+  Row,
+  Tabs,
+} from '@amsterdam/design-system-react'
+import { EditDocumentIcon, EmailIcon } from '@amsterdam/design-system-react-icons'
+import '../common/amopis.css'
```

Comment:
Not a fan of this approach nor the grey background and blue vertical menu. Honestly we should be able to use just the compact tokens and have a header with more dropdown options (for example. user avatar could have a context menu with settings like most webapps). The multiple layers of navigation are not well thought out and I believe UX can improve here.

Reply:
This is the current design for our applications.

I agree that we should discuss using this stylesheet here.

Publishing these application pages isn’t necessarily the goal – evaluating our updated spacing and font sizing is.